src/routes/guides/deploying-your-app.mdx

```diff
@@ -214,8 +213,8 @@ npm i -g firebase-tools
 </div>
 
 <div id="yarn">
-```
-bash yarn global add firebase-tools
+```bash frame="none"
+yarn global add firebase-tools
```

Comment:
Just a heads to be checked: please make sure this doesn't break this rendering. We have had bugs in the past where the autoformatter broke the rendering of this component